tor_ocr/main.py

```diff
@@ -76,8 +97,34 @@ def chunks(s, n):
         yield s[start:(start + n)]
 
 
-def run(config):
+def ocr_space_url(url,
+                  overlay=False,
+                  api_key=os.getenv('OCR_API_KEY', 'helloworld')):
+    """
+    OCR.space API request with remote file.
+    Python3.5 - not tested on 2.7
+    This code is stolen from
+    https://github.com/Zaargh/ocr.space_code_example/blob/master/ocrspace_example.py
+    :param url: Image url.
+    :param overlay: Is OCR.space overlay required in your response.
+        Defaults to False.
+    :param api_key: OCR.space API key.
+        Defaults to environment variable "OCR_API_KEY"
+        If it doesn't exist, it will use "helloworld"
+    :return: Result in JSON format.
+    """
```

Comment:
Love this documentation. Really nice work here!

Reply:
Thanks!